### PR TITLE
Aspdye issue template

### PR DESCRIPTION
**Pull request by ChristophWurst**
squashed version of #1214.

@owncloud/mail review & merge please

### CODE REVIEW

File: issue_template.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+sudo -u www-data php occ config:list system
+from within your ownCloud installation folder
+
+or 
+
+Insert your config.php content here
+(Without the database password, passwordsalt and secret)
+```
+
+**Are you using external storage, if yes which one:** local/smb/sftp/...
```
Comment (jancborchardt) [inline issue_template.md]: This as well as encryption, external user-backend and LDAP as well seems irrelevant for us, no? Sure we need info, but we should also not make it a crazy chore to report issues.

Comment (ChristophWurst) [inline issue_template.md]: True that! I'll remove it

**Comment by jancborchardt** [comment]
In the log section down there we should probably also mention the horde.log again? https://github.com/owncloud/mail#debug-mode

**Comment by jancborchardt** [comment]
Otherwise it’s fine :)

**Comment by ChristophWurst** [comment]
I added the horde log section, so let's get this in.
